pennylane/qnode.py

```diff
@@ -998,16 +998,22 @@ def __call__(self, *args, **kwargs) -> qml.typing.Result:
             )
         # Calculate the classical jacobians if necessary
         if full_transform_program.has_classical_cotransform():
-            argnums = full_transform_program[-1]._kwargs.pop(
-                "argnums", None
-            )  # pylint: disable=protected-access
-            full_transform_program._set_all_classical_jacobians(
-                self, args, kwargs, argnums
-            )  # pylint: disable=protected-access
-            full_transform_program._set_all_argnums(
-                self, args, kwargs, argnums
+            hybrid = full_transform_program[-1]._kwargs.pop(
+                "hybrid", True

```

Comment:
I'm a little lost here. Is the `hybrid` kwarg `True` when we return the classical jacobian and `False` if we return the quantum jacobian?

Reply:
Yes the default in the gradient transform is True, which means we do classical jacobian * quantum jacobian. But you can specify False and you do get the classical jacobian